README.md

```diff
@@ -385,7 +385,7 @@ function `NamedMapper(name, mapper)`.
 
 | Name              | Description
 |-------------------|---------------------------------------------------
-| `path`            | A path. ~ expansion is applied.
+| `path`            | A path. ~ expansion is applied. `-` is accepted for stdout.
```

Comment:
I think perhaps write "- can be used for stdout and will be passed unaltered", just to make it clear that `-` is what the user will receive.

Reply:
Makes sense! I noted the same against `existingfile` where I believe this is also the case.